how-to-device-capture-removal.md

```diff
@@ -0,0 +1,6 @@
+# Creating a PeerConnection without capture devices
+
+Scenarios involving peers without audio or video capture devices should follow these steps.  This includes instances running on a server or an Xbox wihout Kinect.
+
+1. Removing audio capture: WebRtc initialization should send `false` for the first parameter to indicate that audio recording devices won't be used: `Org.WebRtc.WebRtcLib.Setup(queue, false, true)`.

```

Comment:
No longer true -- we pass a struct into the setup instead of true/false. The same true false should be put into struct.

Reply:
I see WebRtcLibConfiguration being passed into Setup() but I'm not seeing the true/false members for A/V capture devices.

Reply:
@robin-raymond I am having the same trouble finding tehse true/false values that Sean referenced in Issue #4 - could you point me to the right place to find the struct you're referencing?

Reply:
Chatted with Robin offline, updated docs properly.